src-electron/sdk/matter.js

```diff
@@ -92,19 +73,20 @@ async function computeStoragePolicyForGlobalAttributes(
     attributes.map(async (attribute) => {
       if (attribute.clusterId == null) {
         forcedExternal = await getForcedExternalStorage(db, attribute.id)
-        if (forcedExternal.byName?.[clusterName]?.includes(attribute.name)) {
-          attribute.storagePolicy =
-            dbEnum.storagePolicy.attributeAccessInterface
-        }
+        forcedExternal.map((option) => {
```

Comment:
.map may be a typo here? Convert to for loop and break out of it once the if condition is satisfied?